xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/test/java/org/xwiki/refactoring/script/RequestFactoryTest.java

```diff
@@ -198,7 +198,7 @@ public void createCopyRequest() throws Exception
             new DocumentReference("code", Arrays.asList("Model", "Entity"), "WebHome");
         CopyRequest copyRequest = requestFactory.createCopyRequest(source, destination);
         assertEquals(Arrays.asList(source), copyRequest.getEntityReferences());
-        assertEquals(destination, copyRequest.getDestination());
+        assertEquals(destination.getLastSpaceReference(), copyRequest.getDestination());
```

Comment:
`destination.getParent()` return the same thing but is more readable in this context I think